XenoHolographic-it/IAR-SDK-Sample-iOS
# IAR-5058 Shared UserManagement

# Overview

Adapted from old sample App's UserController

# Resources

Jira https://iversoft.atlassian.net/browse/IAR-4935
Jira https://iversoft.atlassian.net/browse/IAR-5058

# Technical Considerations

Files on "SharedFiles" are located on the common folder and can be added to the 3 projects if necessary. Avoiding re-writting, but also avoiding creating another library/project/framework.

# Testing Instructions

User Management view should behave similar to the Account view on the old sample App

# Screenshots

![userManagement](https://user-images.githubusercontent.com/9361619/149382141-26ef36c5-ddcf-476a-8140-292ea62493aa.jpg)
